src/pages/WorkspaceSwitcherPage/index.tsx

```diff
@@ -183,7 +182,14 @@ function WorkspaceSwitcherPage() {
                         pressableStyle={styles.flexRow}
                         shouldSyncFocus={false}
                     />
-                    <WorkspacesSectionHeader shouldShowCreateWorkspaceIcon={!shouldShowCreateWorkspace} />
+                    <View style={[styles.ph5, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, styles.mv2]}>
```

Comment:
We only have text, so flex alignment isn’t needed in this case.
```suggestion
                    <View style={[styles.ph5, styles.mv2]}>
```